app/web/modules/stripe/legalEntity.model.js

```diff
@@ -0,0 +1,61 @@
+const Joi = require('joi')
+
+const { ValidationError } = require('./../../../lib/errors')
+const { addEntityIfValid, stripEmpty } = require('./../../../lib/validation')
+const Address = require('./address.model')
+const Dob = require('./dob.model')
+
+const schema = {
+  org_id: Joi.string(),
+  org_name: Joi.string().required(),
+  person_first_name: Joi.string(),
+  person_last_name: Joi.string()
+}
+
+const addSubModels = function (legalEntity, params) {
```

Comment:
Small thing but naming the function here will help in future stack trace woes
```suggestion
const addSubModels = function addSubModels (legalEntity, params) {
```